micropython/usb/examples/device/hid_custom_keypad_example.py

```diff
@@ -0,0 +1,137 @@
+# MicroPython USB HID custom Keypad example
+#
+# This example demonstrates creating a custom HID device with its own
+# HID descriptor, in this case for a USB number keypad.
+#
+# For higher level examples that require less code to use, see mouse_example.py
+# and keyboard_example.py
+#
+# This example uses the usb-device-hid package for the HIDInterface class.
+# This can be installed with:
+#
+# mpremote mip install usb-device-hid
```

Comment:
Suggest adding full (but short) instructions to run the example, as above for the CDC example.